migration/migrating-openshift-3-to-4.adoc

```diff
@@ -0,0 +1,92 @@
+[id="migrating-openshift-3-to-4"]
+= Migrating {product-title} version 3 to 4
+include::modules/common-attributes.adoc[]
+:context: migrating-openshift-3-to-4
+
+toc::[]
+
+You can migrate application workloads from {product-title} 3.7 to a later version with the Cluster Application Migration (CAM) tool. The CAM enables you to control the migration and to minimize application downtime. The CAM provides a web console and an API, based on Kubernetes custom resources, for migrating stateful application workloads, at the granularity of a namespace, from a source to a target cluster.
+
+You can migrate data to any storage class that is available on the target cluster, for example, from Red Hat Gluster Storage or NFS storage to Red Hat Ceph Storage.
+
+Optionally, you can migrate the {product-title} 3.7 (and later) control plane settings to {product-title} 4.x with the Control Plane Migration Assistant (CPMA). See xref:migration-understanding-cpma_{context}[].
+
+.Prerequisites
+
+* You must have `cluster-admin` privileges.
+* You must have `podman` installed.
+* The source cluster(s) must be {product-title} 3.7 or later.
+* The target cluster must be {product-title} 3.9 or later.
+* The replication repository object storage must support the S3 API, for example, Red Hat NooBaa Storage, AWS S3, and generic S3 storage providers.
```

Comment:
I assume we need to change "Red Hat NooBaa Storage".

Upstream:  NooBaa
Downstream: MCG

For this immediate release we will use upstream NooBaa as an example, after OCS4 releases we will change to leverage an operator for MCG.

I will ask product management for their recommendation of what to call this now.

Reply:
Confirmed with product management.

Let's change to:

"The replication repository is an object store endpoint that needs to support the S3 API, for example, NooBaa, AWS S3, and generic S3 storage providers."

Reply:
Do we need to call this out as "upstream" and possibly include a note / call out (warning) about support not being provided for upstream projects?

Reply:
Changed `Red Hat NooBaa Storage` to `NooBaa`.